guides/weka/manifests/modelserver/base/decode.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: decode
+  labels:
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      llm-d.ai/inferenceServing: "true"
+      llm-d.ai/model: Llama-33-70B-Instruct
+      llm-d.ai/role: decode
+  template:
+    metadata:
+      labels:
+        llm-d.ai/inferenceServing: "true"
+        llm-d.ai/model: Llama-33-70B-Instruct
+        llm-d.ai/role: decode
+    spec:
+      initContainers:
+        - name: routing-proxy
+          args:
+            - --port=8000
+            - --vllm-port=8200
+            - --connector=nixlv2
+            - -v=1
+            - --secure-proxy=false
+          image: ghcr.io/llm-d/llm-d-routing-sidecar:v0.2.0
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8000
+          resources: {}
+          restartPolicy: Always
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+        - name: create-cufile-on-node # this init container will create the cufile on the node and runs before vllm
+          image: # OUR DOCKER IMAGE GOES HERE
```

Comment:
I took a look at your `amg-utils` image and its pretty big - the `nvidia` base image itself is about 3.5 Gb. I know there are other utilities in there such as `amg-qad`, `bpftrace`, etc. What is actually the minimum that we need here just to poll the system and create the `cufile` on the host node? It would create a better experience to use a minimally sized container to do this creation, especially because this run quickly at startup, and thats it. So ideally we create a new slim version of the container.

Reply:
Oh yeah I'm 100% creating a new container for this. The image is so big because `amgctl` installs PyTorch+LMCache+vLLM and their dependencies on it. We don't need any of these stuff for this :)